lnwallet/chanfunding/interface.go

```diff
@@ -4,9 +4,11 @@ import (
 	"time"
 
 	"github.com/btcsuite/btcd/btcutil"
+	"github.com/btcsuite/btcd/chaincfg/chainhash"
```

Comment:
nit: this commit says "remote assembler.go..." I think should be "rename assembler.go to.."

Reply:
Fixed!